js/privatebin.js

```diff
@@ -1069,11 +1070,13 @@ jQuery.PrivateBin = function($, sjcl, Base64, RawDeflate) {
 
             // show text
             if (args !== null) {
-                // add jQuery object to it as first parameter
-                args.unshift($element.find(':last'));
-
-                // pass it to I18n
-                I18n._.apply(this, args);
+                // get last text node of element
+                var content = $element.contents();
+                if (content.length > 1) {
+                    content[content.length - 1].nodeValue = ' ' + I18n._(args);
```

Comment:
Thanks for the tip reg. how the lazy loading now is supposed to work. Fixed it for anon comments and vizhash titles.

Reg. spans: I had taken the effort to readd them and waited weeks for you to finally fix the page tpl. You told me you currently have no time, fair enough. So I'll take this over and fix it so it works again, just as before. Spaces aren't broken, they are a feature of HTML. This was how PB has always worked, so how could it suddenly be broken except due to your code change. Not to speak of all the other mess you did not clean up: Passwords only worked with attachments, unit tests for both PHP and JS failing, etc. No time to fix stuff, but time to argue on minutiae. :angry: